docs/docs-content/clusters/public-cloud/azure/azure-disk-encryption.md

```diff
@@ -0,0 +1,353 @@
+
+By default, Azure encrypts all managed disks with
+[platform-managed keys](https://learn.microsoft.com/en-us/azure/virtual-machines/disk-encryption#platform-managed-keys),
+however, customer-managed keys enable you to have greater control over your key management.
+
+You can use Disk Encryption Sets to encrypt your nodes' Operating System (OS) and data disks by including a preset
+option in the <VersionedLink text="Palette eXtended Kubernetes (PXK)" url="/integrations/packs/?pack=kubernetes" />
+pack.
+
+## Limitations
+
+- Azure Disk Encryption is available from [4.5.a](../../../release-notes/release-notes.md) onwards.
+
+- Azure Disk Encryption is only supported on Azure IaaS clusters.
+
+- Azure Disk Encryption is only supported when using the
+
+  <VersionedLink text="Palette eXtended Kubernetes (PXK)" url="/integrations/packs/?pack=kubernetes" /> pack.
+
+- If a key expires in your Key Vault, your cluster may experience operation failures. To resolve this, generate a new
+  key in Key Vault and update your Disk Encryption Set to reference the new key.
+
+  - No changes are needed in Palette, as the Palette eXtended Kubernetes pack configuration references the URI of your
+    Disk Encryption Set, which remains unchanged.
+
+  - We recommend enabling
+    [**Auto key rotation**](https://learn.microsoft.com/en-us/azure/virtual-machines/disk-encryption#automatic-key-rotation-of-customer-managed-keys)
+    on your Disk Encryption Set so it can automatically use new key versions from your Key Vault.
+
+- Changing the Disk Encryption Set URI (`diskEncryptionSetID`) in the Palette eXtended Kubernetes pack configuration in
+  Palette will trigger a node repave.
+
+## Prerequisites
+
+- To create the Azure Key Vault and Disk Encryption Set, you will need an Azure user account with the **Contributor** or
+  **Owner** role on the resource group or subscription.
+
+- Create an
+  [Azure Key Vault](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-customer-managed-keys-portal#set-up-your-azure-key-vault)
+  with the following necessary configuration:
+
+  - Resource access:
+
+    - :white_check_mark: **Azure Virtual Machines for deployment**
+
+  - Purge protection:
+    - :white_check_mark: **Enable purge protection**
+
+- Create a
+  [Disk Encryption Set](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-enable-customer-managed-keys-portal#set-up-your-disk-encryption-set)
+  with the encryption type as **Encryption at-rest with a customer-managed key**.
+
+- In the Azure Key Vault, assign the following
+  [access policies](https://learn.microsoft.com/en-us/azure/key-vault/general/assign-access-policy?tabs=azure-portal) to
+  the Disk Encryption Set that you want to use:
+
+  - Key Management Operations:
+
+    - :white_check_mark: **Get**
+
+  - Cryptographic Operations:
+
+    - :white_check_mark: **Unwrap Key**
+    - :white_check_mark: **Wrap Key**
+
```

Comment:
@benradstone I'm not sure what this is, but it's not our style. The emojis are primarily only reserved for markdown tables with a small set of exceptions.  Consider using a table to list out the required permissions for each resource, or stick with a list. Here is an [example](https://docs.spectrocloud.com/clusters/public-cloud/aws/enable-secrets-encryption-kms-key/) where IAM permissions are specified for a how to.

Reply:
Thank you, I was looking for an example like that.